numpy/core/fromnumeric.py

```diff
@@ -1548,6 +1548,22 @@ def diagonal(a, offset=0, axis1=0, axis2=1):
     array([[1, 3],
            [5, 7]])
 
+    Anti-diagonal example:
+    The anti-diagonal can be obtained by reversing the order of elements
+    using either `numpy.flipud` or `numpy.fliplr`.
+
+    >>> a = np.arange(9).reshape(3, 3)
+    >>> a
+    array([[0, 1, 2],
+           [3, 4, 5],
+           [6, 7, 8]])
+    >>> np.fliplr(a).diagonal() # Horizontal flip
+    array([2, 4, 6])
+    >>> np.flipud(a).diagonal() # Vertical flip
```

Comment:
Note that pep8 wants two spaces before `#`

Reply:
ok,  I just followed the example of the lines 1534, 1535, 1536 in this same page

```python
    >>> a.diagonal(0, # Main diagonals of two arrays created by skipping
    ...            0, # across the outer(left)-most axis last and
    ...            1) # the "middle" (row) axis first.
```

Reply:
You didn't need to go change the other lines, but fine that you did anyway. Thanks!